### ISSUE TITLE
BUG: musi.sh not login in with Safari 13.03 or Firefox 71.0 on Mac OS 10.14.6

### ISSUE THREAD

**Issue by geoperdis** [Musish/Musish#470]
**Describe the bug**

musi.sh/apple.com are not completing Apple ID login after entering double factor authorization code in pop-up window. 
Pop up window remains white and login to musi.sh is not completed.
BUG is present/replicated on both Safari 13.03 or Firefox 71.0 on Mac OS 10.14.6

**To Reproduce**

1. Go to https://musi.sh/
2. Click on Connect to Apple Music
3. Sign in with your Apple ID username and password
4. If you use double factor authentication with your Apple ID, enter the verification code (sent by Apple to one of your devices) in the pop up window.
5. Pop up window turns solid white but login does not complete. 

![enter vercode1](https://user-images.githubusercontent.com/52454/72105200-15137c00-32fb-11ea-9112-0d99cd518a68.png)

![enter vercode2 blank popup](https://user-images.githubusercontent.com/52454/72105309-5572fa00-32fb-11ea-8e89-980f03f5cc3f.png)

**Expected behavior**
After completing step 4 (see above) Apple ID verification is completed and you should be sent back to https://musi.sh/ logged in and continue on to listen to Apple Music.

**Screenshots**

**Desktop 
- OS: Mac OS 10.14.6]
- Browsers: Safari 13.03 or Firefox 71.0

**Comment by BrychanOdlum** [comment]
Strange.

I'm seeing a separate issue this evening where it won't even take me to the 2FA step. The code pops up on my devices, but the auth window tells me my password wasn't correct.

I found that appending the 2FA code to my password in the password input would work. Pretty sure I saw this happen a few months ago too.

I'm assuming someone at Apple broke something for now, will report tomorrow if it's still ongoing.
